custom_components/sat/climate.py

```
@@ -28,32 +29,32 @@
-from homeassistant.core import HomeAssistant, ServiceCall, Event
+from homeassistant.const import ATTR_TEMPERATURE, STATE_UNAVAILABLE, STATE_UNKNOWN, ATTR_ENTITY_ID, STATE_ON, STATE_OFF, EVENT_HOMEASSISTANT_STARTED
+from homeassistant.core import HomeAssistant, ServiceCall, Event, CoreState
 from homeassistant.helpers import entity_registry
 from homeassistant.helpers.entity_platform import AddEntitiesCallback
 from homeassistant.helpers.event import async_track_state_change_event, async_track_time_interval
 from homeassistant.helpers.restore_state import RestoreEntity
 
 from .area import Areas, SENSOR_TEMPERATURE_ID
-from .boiler_state import BoilerState
+from .boiler import BoilerState
 from .const import *
```

Ruff (0.8.2)

`from .const import *` used; unable to detect undefined names

(F403)